src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1892,7 +1892,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <Platforms>$(Platforms)</Platforms>
         <!-- .vcxproj and .nativeproj contain a `ProjectConfiguration` item that have `Platform` metadata within.
              Build the `Platforms` property from that. -->
-        <Platforms Condition="'@(ProjectConfiguration)' != '' and ('$(MSBuildProjectExtension)' == '.vcxproj' or '$(MSBuildProjectExtension)' == '.nativeproj')">@(ProjectConfiguration->'%(Platform)'->Distinct())</Platforms>
+        <Platforms Condition="'@(ProjectConfiguration)' != '' and ('$(MSBuildProjectExtension)' == '.vcxproj' or '$(MSBuildProjectExtension)' == '.nativeproj')">$(Platforms);@(ProjectConfiguration->'%(Platform)'->Distinct())</Platforms>
```

Comment:
This still leaves duplicates here. One solution is to gather all the platforms in a separate item, then de-duplicate it and finally add it here. Seems like overkill but is there any other way? like a property function! maybe?

Reply:
Also, `Win32` should map to `x86` in this case.

Reply:
> This still leaves duplicates here.

true, this is meant to be a "better than it is now, but enables critical scenario" type PR. Without this, the user has no way to define project-level `Platforms` without it being overridden entirely by some import that's still required (because legacy reasons) but not relevant in terms of `Platforms`.

In terms of de-duping, having a `Distinct` for a semicolon-delimited property would be really convenient here, but shouldn't hold back this PR.

double checking, @MIchaelRShea this unblocks you right? 😄

Reply:
Gotcha. Made it into an opt out via setting `ImportProjectConfigurationPlatforms` to false in the referenced project.

At this point it doesn't sound like we need it to append to $(Platforms) anymore right? Unless you can think of some scenario where this is useful.

Reply:
I agree I don't think it needs to bring in platform, cant think of a scenario where both is needed since platforms can always be updated to include everything needed.